diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ae4fddb..e5aeb12 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -80,7 +80,7 @@ if(APPLE)
   set(MACOSX_BUNDLE_COPYRIGHT "2003- The KDiff3 Authors")
 endif()
 
-if(ENABLE_GDBINDEX AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND ${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+if(ENABLE_GDBINDEX AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     find_program(GDBINDEX "gdb-add-index")
     if(GDBINDEX)
         add_custom_command(TARGET kdiff3 POST_BUILD
